{
    "affected": [
        {
            "package": {
                "ecosystem": "photon:1.0",
                "name": "git",
                "purl": "pkg:rpm/vmware/git?distro=photon-1"
            }
        },
        {
            "package": {
                "ecosystem": "photon:1.0",
                "name": "linux",
                "purl": "pkg:rpm/vmware/linux?distro=photon-1"
            }
        },
        {
            "package": {
                "ecosystem": "photon:1.0",
                "name": "openldap",
                "purl": "pkg:rpm/vmware/openldap?distro=photon-1"
            }
        },
        {
            "package": {
                "ecosystem": "photon:1.0",
                "name": "glibc",
                "purl": "pkg:rpm/vmware/glibc?distro=photon-1"
            }
        },
        {
            "package": {
                "ecosystem": "photon:1.0",
                "name": "linux-esx",
                "purl": "pkg:rpm/vmware/linux-esx?distro=photon-1"
            }
        }
    ],
    "id": "PHSA-2020-0293",
    "modified": "2023-03-16T05:25:30Z",
    "published": "2020-05-12T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://github.com/vmware/photon/wiki/Security-Update-1.0-293"
        }
    ],
    "related": [
        "CVE-2020-11008",
        "CVE-2020-12243",
        "CVE-2016-10739",
        "CVE-2020-10711"
    ]
}